employees/models.py

```diff
@@ -59,7 +59,7 @@ class Report(models.Model):
     creation_date = models.DateTimeField(auto_now_add=True)
     last_update = models.DateTimeField(auto_now=True)
     author = models.ForeignKey(CustomUser, on_delete=models.PROTECT)
-    project = models.ForeignKey(Project, on_delete=models.CASCADE)
+    project = models.ForeignKey(Project, on_delete=models.CASCADE, blank=False, default=1)
```

Comment:
Is `default` really needed? I think it might cause problems because you are pointing to ID here that might not necessary exist, ie. on fresh installation or when object with ID 1 was removed. `blank=False`.